includes/polling/wifi.inc.php

```diff
@@ -7,6 +7,8 @@
     } elseif ($device['os'] == 'airos-af') {
         echo 'It Is AirFIBER' . PHP_EOL;
         include 'includes/polling/mib/ubnt-airfiber-mib.inc.php';
+        echo 'It Is AirFIBER - Rx' . PHP_EOL;
+        include 'includes/polling/wireless/ubnt-airfibre-rx.inc.php';
```

Comment:
@laf I agree the rx code should go in the airfibre-mib file.

I think a notification would be the best approach and see if there are objections. I think there are few, if any, installations graphing airfibre and we are just making extra work trying to keep things simple (for an non-existent user)